web/server.ts

```diff
@@ -38,7 +38,7 @@ app.use((req, res, next) => {
 });
 
 // Parse Cookies and JSON Bodies:
-app.use(bodyParser.json());
+app.use(bodyParser.json({ limit: '1mb' }));
```

Comment:
The default is 100kb. I had to increase it, because the payload of the issue reporter is pretty big.